src/ray/raylet/node_manager.cc

```diff
@@ -953,7 +1004,7 @@ void NodeManager::ProcessDisconnectClientMessage(const WorkerID &worker_id,
 
     // Since some resources may have been released, we can try to dispatch more tasks.
     DispatchTasks(local_queues_.GetReadyTasksWithResources());
-  } else if (is_driver) {
```

Comment:
why removing this?

Reply:
Not used, a worker can be a worker or a driver.